Description: Disable reporting by default
 This patch disables reporting by default. For reporting to be enabled,
 it has to be specified in the config file. This changes the default
 behaviour where reporting would be enabled if influxd is launched
 without specifying a config file.
Author: Alexandre Viau <alexandre@alexandreviau.net>
Last-Update: 2015-08-11

---
 cmd/influxd/run/config.go |    2 +-
 cmd/influxd/run/server.go |    6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/cmd/influxd/run/config.go
+++ b/cmd/influxd/run/config.go
@@ -56,7 +56,7 @@
 	ContinuousQuery continuous_querier.Config `toml:"continuous_queries"`
 
 	// Server reporting
-	ReportingDisabled bool `toml:"reporting-disabled"`
+    ReportingEnabled bool `toml:"reporting-enabled"`
 
 	// BindAddress is the address that all TCP services use (Raft, Snapshot, Cluster, etc.)
 	BindAddress string `toml:"bind-address"`
--- a/cmd/influxd/run/server.go
+++ b/cmd/influxd/run/server.go
@@ -78,7 +78,7 @@
 	Monitor *monitor.Monitor
 
 	// Server reporting and registration
-	reportingDisabled bool
+    reportingEnabled bool
 
 	// Profiling
 	CPUProfile string
@@ -147,7 +147,7 @@
 
 		MetaClient: meta.NewClient(c.Meta),
 
-		reportingDisabled: c.ReportingDisabled,
+		reportingEnabled: c.ReportingEnabled,
 
 		httpAPIAddr: c.HTTPD.BindAddress,
 		httpUseTLS:  c.HTTPD.HTTPSEnabled,
@@ -434,7 +434,7 @@
 	}
 
 	// Start the reporting service, if not disabled.
-	if !s.reportingDisabled {
+	if s.reportingEnabled {
 		go s.startServerReporting()
 	}
 
--- a/cmd/influxd/run/server_helpers_test.go
+++ b/cmd/influxd/run/server_helpers_test.go
@@ -229,7 +229,7 @@
 func NewConfig() *run.Config {
 	c := run.NewConfig()
 	c.BindAddress = "127.0.0.1:0"
-	c.ReportingDisabled = true
+	c.ReportingEnabled = false
 	c.Coordinator.WriteTimeout = toml.Duration(30 * time.Second)
 	c.Meta.Dir = MustTempFile()
 
